cmd/otelcontribcol/components.go

```diff
@@ -4,26 +4,20 @@ package main
 
 import (
 	"go.opentelemetry.io/collector/connector"
-	forwardconnector "go.opentelemetry.io/collector/connector/forwardconnector"
```

Comment:
This file is autogenerated and should have no changes of this kind, could you revert the changes to `components.go` and `components_test.go`?